### PR TITLE
Add webxr platform

### PR DESCRIPTION
**Pull request by StephenHodgson**
# XRTK - Mixed Reality Toolkit Pull Request

## Overview

Adds [WebXR](https://github.com/XRTK/WebXR) platform to the core project

### CODE REVIEW

File: XRTK-Core/Assets/XRTK.Generated/SymbolicLinkSettings.asset [hunk 1]
```diff
@@ -82,3 +82,12 @@ MonoBehaviour:
   - sourceRelativePath: /Submodules/glTF/XRTK.glTF/Packages/com.xrtk.gltf
     targetRelativePath: /XRTK-Core/Packages/com.xrtk.gltf
     isActive: 1
+  - sourceRelativePath: /Submodules/WebXR/XRTK.WebXR/Packages/com.xrtk.webxr
+    targetRelativePath: /XRTK-Core/Packages/com.xrtk.webxr
```
Comment (StephenHodgson) [inline XRTK-Core/Assets/XRTK.Generated/SymbolicLinkSettings.asset:86]:
```suggestion
    targetRelativePath: /XRTK-Core/Packages/com.xrtk.webxr.git
```

**Comment by SimonDarksideJ** [comment]
Still looks good, but don't we need the WebXR platform to build and run first?

**Comment by StephenHodgson** [comment]
@SimonDarksideJ [it is already running builds against WebXR](https://dev.azure.com/xrtk/Mixed%20Reality%20Toolkit/_build/results?buildId=6817&view=logs&jobId=0ab14b9f-e499-56d5-97b1-fd98b70ea339&j=0ab14b9f-e499-56d5-97b1-fd98b70ea339&t=88e774dc-252d-5c6d-8529-f86a4aebeef7)

**Comment by SimonDarksideJ** [comment]
PR does have a module and submodule for WebXR, but the build reports is as a failure and cannot build?

File: .gitmodules [hunk 2]
```diff
@@ -24,4 +24,7 @@
 	url = https://github.com/XRTK/SpatialPersistence.git
 [submodule "Submodules/SpatialPersistence.ASA"]
 	path = Submodules/SpatialPersistence.ASA
-	url = https://github.com/XRTK/SpatialPersistence.ASA.git
\ No newline at end of file
+	url = https://github.com/XRTK/SpatialPersistence.ASA.git
+[submodule "Submodules/WebXR"]
+	path = Submodules/WebXR
+	url = https://github.com/XRTK/com.xrtk.webxr
```
Comment (StephenHodgson) [inline .gitmodules:30]:
```suggestion
	url = https://github.com/XRTK/com.xrtk.webxr.git
```

**Pull request closed without merging**